src/NUnitFramework/tests/Assertions/CollectionAssertTest.cs

```diff
@@ -322,8 +322,8 @@ public void AreNotEqual()
         [Test]
         public void AreNotEqual_Fails()
```

Comment:
This counter part test also fails now, since Object.ReferenceEquals( set1, set2 ) == false, but the items are the same.